icubam/www/handlers/db.py

```diff
@@ -46,6 +48,9 @@ def initialize(self, config, db_factory):
   def get(self, collection):
     file_format = self.get_query_argument('format', default=None)
     max_ts = self.get_query_argument('max_ts', default=None)
+    should_preprocess = (
```

Comment:
Maybe sprinkle a couple comments as this might not be obvious to people down the road.

Reply:
done.